helm/estafette-vulnerability-scanner/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "estafette-vulnerability-scanner.fullname" . }}
+  labels:
+  {{- include "estafette-vulnerability-scanner.labels" . | nindent 6 }}
+spec:
+  endpoints:
+    - path: /metrics
+      port: metrics
+  namespaceSelector:
+    matchNames:
+      - "{{ .Release.Namespace }}"
+  selector:
+    matchLabels:
+  {{- include "estafette-vulnerability-scanner.labels" . | nindent 6 }}
+  {{- end }}
```

Comment:
Please fix the indentation here.

Reply:
Do you mean the `  {{- end }}`? Do you want the leading spaces to be removed?

That's how IntelliJ formatted it for me, but I can change it of course